From 4330a30eee65f8cd6d069efe5b40cf214753971f Mon Sep 17 00:00:00 2001
From: Dave Airlie <airlied@redhat.com>
Date: Thu, 25 Feb 2010 14:05:09 +1000
Patch-mainline: 2.6.35
References: fate#310916
Git-commit: 9fd1de52945e06ed88a440c99ca92dab74b9b33c
Subject: [PATCH 0722/2588] fb: for framebuffer handover don't exit the loop
 early.

An offb machine has been seen in the fields which adds two
offb nodes, so continue scanning the list after removing one.

Signed-off-by: Dave Airlie <airlied@redhat.com>
(cherry picked from commit 9fd1de52945e06ed88a440c99ca92dab74b9b33c)

Signed-off-by: Takashi Iwai <tiwai@suse.de>
---
 drivers/video/fbmem.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/drivers/video/fbmem.c b/drivers/video/fbmem.c
index 99bbd28..a15b44e 100644
--- a/drivers/video/fbmem.c
+++ b/drivers/video/fbmem.c
@@ -1513,7 +1513,6 @@ register_framebuffer(struct fb_info *fb_info)
 				       fb_info->fix.id,
 				       registered_fb[i]->fix.id);
 				unregister_framebuffer(registered_fb[i]);
-				break;
 			}
 		}
 	}
-- 
1.7.6

